Makefile

```diff
@@ -58,6 +58,10 @@ ifeq ($(GO),)
   $(error Could not find 'go' in path.  Please install go, or if already installed either add it to your path or set GO to point to its directory)
 endif
 
+GO_TEST := go test
```

Comment:
just wondering, did you break these out so you can replace with `GO111MODULES=on go test` later? I think I saw this in an earlier version which is why I am wondering if there is any benefit vs just doing `GO111MODULES ?= on` at the top level

Reply:
Yes, it's just to let me enable go modules selectively.

Reply:
Removed.